styles.css

```diff
@@ -1 +1,143 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+body {
+    background-color: navajowhite;
```

Comment:
Hsl en Hex kleur zijn beter om te plaatsen in je code.(ik heb zelf ook heel snel net als jij gedaan hoor)